pages/common/flarectl.md

```diff
@@ -0,0 +1,21 @@
+# flarectl 
+
+> Official CLI for Cloudflare
+> 
+> More information: <https://github.com/cloudflare/cloudflare-go/blob/master/cmd/flarectl/README.md>.
+
+- Block a bad actor's IP via the IP Firewall:
```

Comment:
```suggestion
- Block an IP via the IP Firewall:
```
I think this would be better as mentioned [here](https://github.com/cloudflare/cloudflare-go/blob/master/cmd/flarectl/README.md#block-an-ip-via-the-ip-firewall) on the Github page. It's just my suggestion that we no need to mention `bad actor` in the comment description.